pandas/_libs/tslibs/nattype.pyx

```diff
@@ -713,7 +713,14 @@ NaT = c_NaT        # Python-visible
 
 cdef inline bint checknull_with_nat(object val):
     """ utility to check if a value is a nat or not """
-    return val is None or util.is_nan(val) or val is c_NaT
+    cdef:
```

Comment:
can you move this change to util.is_nan?

Reply:
Sure, will do.